src/dune_rules/melange/melange_stanzas.ml

```diff
@@ -81,16 +102,15 @@ module Emit = struct
          in
          field "target" (plain_string (fun ~loc s -> of_string ~loc s))
        and+ alias = field_o "alias" Alias.Name.decode
-       and+ module_system =
-         field "module_system"
-           (enum [ ("es6", Melange.Module_system.Es6); ("commonjs", CommonJs) ])
+       and+ module_systems =
+         field "module_systems" module_systems
+           ~default:[ Melange.Module_system.default ]
```

Comment:
If we add default now, will we get trapped down the line if we ever want to change it?

Reply:
I think that can probably be done safely if we bump the melange extension to e.g. `0.2`